Tests/Test.Eyes.Selenium.DotNet/VisualGridTests/TestVisualGridApi.cs

```diff
@@ -86,14 +86,14 @@ public override void SetRenderInfo(RenderingInfo renderingInfo)
             }
         }
 
-        private class MockEyesConnectorFactory : IEyesConnectorFactory
-        {
-            public IUfgConnector CreateNewEyesConnector(Logger logger, RenderBrowserInfo browserInfo, Applitools.Configuration config)
-            {
-                logger.Verbose($"creating {nameof(MockEyesConnector)}");
-                return new MockEyesConnector(logger, browserInfo, config);
-            }
-        }
+        //private class MockEyesConnectorFactory : IEyesConnectorFactory
```

Comment:
remove?

Reply:
not yet, as this file entirely is set not to be compiled and it has test we would like to run.